### ISSUE TITLE
My Current Goals Page

### ISSUE THREAD

**Issue by steffipravasi** [bcgov/performance#1119]
https://bcgov.sharepoint.com/:w:/r/teams/02915/_layouts/15/Doc.aspx?action=editNew&sourcedoc=%7B45ffd269-ec29-4070-bf49-6045eaf2e24c%7D&wdOrigin=TEAMS-ELECTRON.teamsSdk.openFilePreview&wdExp=TEAMS-CONTROL&wdhostclicktime=1695190693481&web=1

Reference for checklist  - #1263

**Comment by jizhaogit** [comment]
the status change dropdown still has issue that can not be accessed by keyboard.

**Comment by steffipravasi** [comment]
May 2 Issues – 1,2, 4, 5, 6, 7, 9, 10,  12, 15, 16, 17, 21, 23, 24, 25

**Comment by steffipravasi** [comment]
@jizhaogit I've ignored the elements we discussed today from this ticket and also verified the fixes made. We'll still need to fix the issues May 9 – 4, 6, 15, 23, 24

**Comment by jizhaogit** [comment]
@steffipravasi  Please help move #23, 24 to goal bank related accessibility list.  the modal is trigged in the goal bank page. will take care of it when we work on the goal bank part. thank you!

**Comment by jizhaogit** [comment]
@steffipravasi May 14, would you please help verify the latest pushing.  thank you!

**Comment by jizhaogit** [comment]
@steffipravasi  May 16,  the tab change is ready in test.

**Comment by jizhaogit** [comment]
The data picker not auto update issue is addressed.

**Comment by steffipravasi** [comment]
As discussed in the meeting, putting this ticket back to In Progress to work on the date picker:
Issues to be worked on:
i) Once the start and end date are entered, and the user tries to clear the field by removing the date through the keyboard, the dates are added back to the fields
ii) The fields do not clear on the Reset button
iii) We need to finalize a date format which the start date and end date will accept correctly. It is better if the field accepts dd/mm/yyyy or mm/dd/yyyy
iv) If the field is empty, placeholder inside the field must be mm/dd/yyyy or dd/mm/yyyy.
If the placeholder is difficult, we can add a text like this at the bottom of the field or besides the title
<img width="384" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMThQQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--bf8ba5571f7e6f53364358ec30257f1d3c4d0041/Screenshot%202024-05-24%20at%203.45.37%E2%80%AFPM.png" alt="Screenshot 2024-05-24 at 3.45.37 PM.png" />
<img width="436" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMkFQQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--29b824ebd77eef81354cdcb873ef3bc5fba209a2/Screenshot%202024-05-24%20at%203.45.32%E2%80%AFPM.png" alt="Screenshot 2024-05-24 at 3.45.32 PM.png" />

**Comment by jizhaogit** [comment]
request should be addressed.

**Comment by steffipravasi** [comment]
Start and end date fields issue addressed
